src/vscode-dts/vscode.proposed.chatReferenceBinaryData.d.ts

```diff
@@ -20,10 +20,17 @@ declare module 'vscode' {
 
 		/**
 		 * Retrieves the binary data of the reference. This is primarily used to receive image attachments from the chat.
+		 * Note: base64 will only be used as a fallback if upload fails.
 		 * @returns A promise that resolves to the binary data as a Uint8Array.
 		 */
 		data(): Thenable<Uint8Array>;
 
+		/**
+		 * URL of image that was uploaded to GitHub
+		 */
+		url: string;
```

Comment:
These two are really an either or right? Either there is data or it has been uploaded to GitHub?

I think API should reflect this. API Sync will probably agree.